pod/perlfunc.pod

```diff
@@ -5932,7 +5932,7 @@ L<C<unpack>|/unpack TEMPLATE,EXPR>, also via the C</>
 template character.  Within each repetition of a group, positioning with
 C<@> starts over at 0.  Therefore, the result of
 
-    pack("@1A((@2A)@3A)", qw[X Y Z])
+    pack('@1A((@2A)@3A)', qw[X Y Z])
```

Comment:
We've already had someone who I'd expect to know better mess up the quotes here, maybe it could use a comment warning about that.

Reply:
Good point. Added.